stubs/python-decouple/decouple.pyi

```diff
@@ -0,0 +1,158 @@
+
+class _Repository(Protocol):  # undocumented
+    def __contains__(self, key: str) -> bool: ...
+    def __getitem__(self, key: str) -> str: ...
+
+class RepositoryEmpty:  # undocumented
+    def __init__(self, source: str = ..., encoding: str = ...) -> None: ...
+    def __contains__(self, key: str) -> bool: ...
+    def __getitem__(self, key: str) -> str: ...
+
+class RepositoryIni(RepositoryEmpty):
+    SECTION: str
+    parser: ConfigParser
+    def __init__(self, source: str, encoding: str = ...) -> None: ...
+    def __contains__(self, key: str) -> bool: ...
+    def __getitem__(self, key: str) -> str: ...
+
+class RepositoryEnv(RepositoryEmpty):
+    data: dict[str, str]
+    def __init__(self, source: str, encoding: str = ...) -> None: ...
+    def __contains__(self, key: str) -> bool: ...
+    def __getitem__(self, key: str) -> str: ...
+
+class RepositorySecret(RepositoryEmpty):
+    data: dict[str, str]
+    def __init__(self, source: str = ...) -> None: ...
+    def __contains__(self, key: str) -> bool: ...
+    def __getitem__(self, key: str) -> str: ...
+
+class AutoConfig:
+    SUPPORTED: OrderedDict[str, RepositoryEmpty]
+    encoding: str
+    search_path: str | None
+    config: Config | None
+    def __init__(self, search_path: str | None = None) -> None: ...
+    @overload
+    def __call__(self, option: str, default: str = ...) -> str: ...
+    @overload
+    def __call__(self, option: str, *, cast: Callable[[str], _T]) -> _T: ...
+    @overload
+    def __call__(self, option: str, default: str, cast: Callable[[str], _T]) -> _T: ...
+
+config: AutoConfig
+
+class Csv(Generic[_T, _TCsv]):
+    cast: Callable[[str], _T]
+    delimiter: str
+    strip: str
+    post_process: Callable[[Iterable[Any]], _TCsv]
+    def __call__(self, value: str) -> _TCsv: ...
+    @overload
+    def __init__(
+        self: Csv[str, list[str]],
+        cast: Callable[[str], str] = ...,
+        delimiter: str = ...,
+        strip: str = ...,
+        post_process: Callable[[Iterable[Any]], list[str]] = ...,
+    ) -> None: ...
+    @overload
+    def __init__(
+        self: Csv[_T, list[_T]],
+        cast: Callable[[str], _T],
+        delimiter: str = ...,
+        strip: str = ...,
+        post_process: Callable[[Iterable[_T]], list[_T]] = ...,
+    ) -> None: ...
+    @overload
+    def __init__(
+        self: Csv[str, _TCsv],
+        cast: Callable[[str], str] = ...,
+        delimiter: str = ...,
+        strip: str = ...,
+        *,
+        post_process: Callable[[Iterable[Any]], _TCsv],
+    ) -> None: ...
+    @overload
+    def __init__(
+        self: Csv[_T, _TCsv],
+        cast: Callable[[str], _T],
+        delimiter: str = ",",
+        strip: str = ...,
+        *,
+        post_process: Callable[[Iterable[Any]], _TCsv],
+    ) -> None: ...
+    @overload
+    def __init__(
+        self: Csv[_T, _TCsv], cast: Callable[[str], _T], delimiter: str, strip: str, post_process: Callable[[Iterable[_T]], _TCsv]
```

Comment:
```suggestion
        post_process: Callable[[Iterable[Any]], list[_T]] = ...,
    ) -> None: ...
    @overload
    def __init__(
        self: Csv[str, _TCsv],
        cast: Callable[[str], str] = ...,
        delimiter: str = ...,
        strip: str = ...,
        *,
        post_process: Callable[[Iterable[Any]], _TCsv],
    ) -> None: ...
    @overload
    def __init__(
        self: Csv[_T, _TCsv],
        cast: Callable[[str], _T],
        delimiter: str = ",",
        strip: str = ...,
        *,
        post_process: Callable[[Iterable[Any]], _TCsv],
    ) -> None: ...
    @overload
    def __init__(
        self: Csv[_T, _TCsv], cast: Callable[[str], _T], delimiter: str, strip: str, post_process: Callable[[Iterable[Any]], _TCsv]
```
You missed replacing a couple of `Iterable[_T]` with `Iterable[Any]`

Reply:
Thanks, updated!